src/main/resources/liquibase/crop_changelog/21_3_0.xml

```diff
@@ -0,0 +1,814 @@
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT count(1) FROM cvtermprop WHERE cvterm_id = 8230 AND type_id = 1800 AND value = 'Germplasm Descriptor';
+			</sqlCheck>
+		</preConditions>
+		<comment>Change 'ENTRY_NO' variable type to entry detail</comment>
+		<sql dbms="mysql" splitStatements="true">
+			UPDATE cvtermprop SET value = 'Entry Detail' WHERE cvterm_id = 8230 AND type_id = 1800 AND value = 'Germplasm Descriptor';
+
+			INSERT INTO entry_detail_migration_output (cvterm_id) VALUES (8230);
+		</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="v21.3.0-15">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT count(1) > 0 FROM list_data_view WHERE cvterm_id = 8230 and type_id = 1815;
+			</sqlCheck>
+		</preConditions>
+		<comment>Migrate ENTRY_NO as an Entry Detail variable</comment>
+		<sql dbms="mysql" splitStatements="true">
+			<!-- Remove ENTRY_NO as default view -->
+			DELETE FROM list_data_default_view WHERE name = 'ENTRY_NO';
+
+			<!-- Remove ENTRY_NO as static column -->
+			DELETE FROM list_data_view WHERE static_id = 8230;
+
+			<!-- Insert ENTRY_NO variable to list_data_view -->
+			INSERT INTO list_data_view(listid,cvterm_id,type_id)
+			SELECT distinct(ld.listid), 8230, 1815 FROM listnms ld;
+		</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="v21.3.0-16">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT count(1) > 0 FROM projectprop WHERE variable_id = 8230 AND type_id = 1804;
+            </sqlCheck>
+        </preConditions>
+        <comment>Change 'ENTRY_NO' variable type to entry detail in 'projectprop' table</comment>
+        <sql dbms="mysql" splitStatements="true">
+			UPDATE projectprop SET type_id = 1815 WHERE variable_id = 8230 AND type_id = 1804;
+        </sql>
+	</changeSet>
+	<!-- End of ENTRY_NO related changesets-->
+
+	<!-- SEED_SOURCE related changesets-->
+	<changeSet author="cuenyad" id="v21.3.0-17">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT count(1) > 0 FROM projectprop WHERE variable_id = 8360
+            </sqlCheck>
+        </preConditions>
+        <comment>Replace the SEED_SOURCE variable by ENTRY_SOURCE IN projectprop AND stockprop</comment>
+        <sql dbms="mysql" splitStatements="true">
+			set @ENTRY_SOURCE_ID = (select cvterm_id from cvterm where name ='ENTRY_SOURCE' and cv_id = 1040 and is_obsolete = 0);
+
+			# Replace SEED_SOURCE variable by ENTRY_SOURCE
+			update projectprop set variable_id = @ENTRY_SOURCE_ID , alias = 'ENTRY_SOURCE', type_id = 1815
+			where variable_id = 8360 and type_id = 1804;
+			update projectprop set alias = 'ENTRY_SOURCE' where alias = 'SOURCE' and variable_id = 8360;
+
+			# Update cvterm_id 8360 from (SEED_SOURCE) to ENTRY_SOURCE cvterm_id
+			update stockprop set type_id = @ENTRY_SOURCE_ID where type_id = 8360;
```

Comment:
@cuenyad let's check this tomorrow. Seems to me that is ok but let's do a quick test tomorrow